packages/react/src/components/FluidTimePicker/__tests__/FluidTimePicker-test.js

```diff
@@ -60,5 +60,81 @@ describe('FluidTimePicker', () => {
     });
+
+    it('returns invalidText when invalid is true', async () => {
+      render(
+        <FluidTimePicker
+          className="custom-class"
+          id="time=picker-1"
+          data-testid="timePicker-1"
+          labelText="Time"
+          invalid
+          invalidText="Invalid text">
+          <FluidTimePickerSelect id="select-1" labelText="Clock">
+            <SelectItem value="am" text="AM" />
+            <SelectItem value="pm" text="PM" />
+          </FluidTimePickerSelect>
+          <FluidTimePickerSelect id="select-2" labelText="Timezone">
+            <SelectItem value="et" text="Eastern Time (ET)" />
+            <SelectItem value="ct" text="Central Time (CT)" />
+            <SelectItem value="mt" text="Mountain Time (MT)" />
+            <SelectItem value="pt" text="Pacific Time (PT)" />
+          </FluidTimePickerSelect>
+        </FluidTimePicker>
+      );
+      await expect(screen.getByText('Invalid text')).toBeInTheDocument();
+    });
+
+    it('returns warnText when warn is true', async () => {
+      render(
+        <FluidTimePicker
+          className="custom-class"
+          id="time=picker-1"
+          data-testid="timePicker-1"
+          labelText="Time"
+          warn
+          warnText="Warn text">
+          <FluidTimePickerSelect id="select-1" labelText="Clock">
+            <SelectItem value="am" text="AM" />
+            <SelectItem value="pm" text="PM" />
+          </FluidTimePickerSelect>
+          <FluidTimePickerSelect id="select-2" labelText="Timezone">
+            <SelectItem value="et" text="Eastern Time (ET)" />
+            <SelectItem value="ct" text="Central Time (CT)" />
+            <SelectItem value="mt" text="Mountain Time (MT)" />
+            <SelectItem value="pt" text="Pacific Time (PT)" />
+          </FluidTimePickerSelect>
+        </FluidTimePicker>
+      );
+      await expect(screen.getByText('Warn text')).toBeInTheDocument();
+    });
+
+    it('should set disabled as expected', () => {
+      const onClick = jest.fn();
+
+      render(
+        <FluidTimePicker
+          className="custom-class"
+          id="time=picker-1"
+          data-testid="timePicker-1"
+          labelText="Time"
+          onClick={onClick}
+          disabled>
+          <FluidTimePickerSelect id="select-1" labelText="Clock">
+            <SelectItem value="am" text="AM" />
+            <SelectItem value="pm" text="PM" />
+          </FluidTimePickerSelect>
+          <FluidTimePickerSelect id="select-2" labelText="Timezone">
+            <SelectItem value="et" text="Eastern Time (ET)" />
+            <SelectItem value="ct" text="Central Time (CT)" />
+            <SelectItem value="mt" text="Mountain Time (MT)" />
+            <SelectItem value="pt" text="Pacific Time (PT)" />
+          </FluidTimePickerSelect>
+        </FluidTimePicker>
+      );
+
+      fireEvent.click(screen.getByRole('textbox'));
```

Comment:
Could this use `userEvent` instead? It more realistically simulates the interaction, [see here](https://github.com/testing-library/eslint-plugin-testing-library/blob/main/docs/rules/prefer-user-event.md)